muskathlon/models/event_compassion.py

```diff
@@ -85,3 +86,14 @@ def get_muskathlon_participants(self, event_id):
                 'country': participant.partner_id.country_id.name
             })
         return ret
+
+    @api.multi
```

Comment:
CRON methods are static
```suggestion
    @api.model
```